app/src/main/res/transition/image_shared_element_transition.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<transitionSet
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="375"
+    android:interpolator="@android:interpolator/fast_out_slow_in"
+    android:transitionOrdering="together">
+    <changeClipBounds/>
+    <changeTransform/>
+    <changeBounds/>
+</transitionSet>
```

Comment:
Add newline to the end of the file.

```suggestion
</transitionSet>

```

Also, you can set “Ensure line feed at file end on Save” in Android Studio under Editor > General in Preferences to avoid this in the future.

Reply:
Thanks for the tip, and the review! :)